main.go

```diff
@@ -249,6 +257,18 @@ func start_app(config Config) {
 
 	http.Handle(static_uri, http.StripPrefix(static_uri, fs))
 
+	// Setup async auth service and build routes
+	stg := memstorage.New(false)
+	gc := storage.NewGC(&stg, time.Minute, false, debug)
+	if err := gc.Start(); err != nil {
+		log.Fatalf("error starting GC for async auth: %v", err)
+	}
+
+	// Hack(jr): We only support one provider in konvoy and it is not necessary to identify
+	// cluster scope in this context (all clusters should use the same provider).
```

Comment:
we we want to add a "tech debt" jira to the backlog to change this?

Reply:
I think this whole repository is tech debt :) We're likely going to replace it soon. I'll open up a card to start a discussion on this.